pages/dmp_knowledge_base/0_admin_information.md

```diff
@@ -33,19 +38,32 @@ not covered
 - planning post-award, final reporting
 ```
 
+#### [RDA Common Standard for maDMP](http://doi.org/10.15497/rda00039) [![License: Unlicense](https://img.shields.io/badge/license-Unlicense-blue.svg)](http://unlicense.org/)
+[Properties in dmp]
+- [#dmp_contact_table](https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard?tab=readme-ov-file#dmp_contact_table): contact_id (identifier, type), mbox, name
+- [#dmp_contributor_table](https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard?tab=readme-ov-file#dmp_contributor_table): contributor_id (identifier, type), mbox, name, role [string, not controlled])
+- [#project_table](https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard?tab=readme-ov-file#project_table): start, end, title, description [string, free text], funding (funder_id [CrossRef funder registry recommended], funding_status, grant_id)
+
+Missing:
+- institution(s) with ROR
```

Comment:
will likely be fixed in next release https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard/pull/71